microcosm_pubsub/tests/test_matcher.py

```diff
@@ -37,7 +37,7 @@ def test_publish_no_messages(self):
         )
 
     def test_publish_one_message(self):
-        self.graph.sns_producer.produce(created("foo"), data="data")
+        self.graph.sns_producer.produce(created("foo"), data="data", uri="http://some-uri")
```

Comment:
These are registered as URIMessages; we were probably ignoring validation earlier.